### ISSUE TITLE
Connection to MPD seems to flicker while logs complain about a broken pipe

### ISSUE THREAD

**Issue by StephenWetzel** [yktoo/ymuse#88]
I just installed ymuse 0.22, and when trying to connect to my mpd server, running on my local machine, I can't seem to get a solid connection.  The track bar and time played/total time seems to work mostly, but occasionally goes blank, however all other info (title, artist, album, etc) is mostly blank, and only occasionally flickers.  The same thing is true of the Library tab, I see my library folders occasionally flash into place, but mostly it's blank.

I am able to connect to and control mpd with both mpc and gmpc.

The same behavior happens when I try to connect to mpd via TCP or socket.  Below are a sample of the logs I see.  Please let me know if there is anything else I can provide that would help.

```
2024/07/22 01:37:43 01:37:43.404 WARNING player PlaylistInfo() failed: EOF
2024/07/22 01:37:43 01:37:43.405 WARNING player updateLibrary(): ListInfo() failed: EOF
2024/07/22 01:37:43 01:37:43.405 WARNING player CurrentSong() failed: write tcp 127.0.0.1:39956->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:43 01:37:43.407 WARNING player PlaylistInfo() failed: write tcp 127.0.0.1:39956->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:43 01:37:43.407 WARNING player updateLibrary(): ListInfo() failed: write tcp 127.0.0.1:39956->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:43 01:37:43.408 WARNING player CurrentSong() failed: write tcp 127.0.0.1:39956->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:44 01:37:44.347 WARNING player Connection to MPD lost
2024/07/22 01:37:44 01:37:44.347 WARNING player Failed to connect to MPD: Status() failed: write tcp 127.0.0.1:39956->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:44 01:37:44.382 WARNING player PlaylistInfo() failed: EOF
2024/07/22 01:37:44 01:37:44.383 WARNING player updateLibrary(): ListInfo() failed: EOF
2024/07/22 01:37:44 01:37:44.383 WARNING player CurrentSong() failed: write tcp 127.0.0.1:39978->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:44 01:37:44.384 WARNING player PlaylistInfo() failed: write tcp 127.0.0.1:39978->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:44 01:37:44.385 WARNING player updateLibrary(): ListInfo() failed: write tcp 127.0.0.1:39978->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:44 01:37:44.385 WARNING player CurrentSong() failed: write tcp 127.0.0.1:39978->127.0.0.1:6600: write: broken pipe
2024/07/22 01:37:45 01:37:45.348 WARNING player Connection to MPD lost
2024/07/22 01:37:45 01:37:45.348 WARNING player Failed to connect to MPD: Status() failed: write tcp 127.0.0.1:39978->127.0.0.1:6600: write: broken pipe
```

And when trying to connect via socket it's similar
```
2024/07/22 01:41:59 01:41:59.529 WARNING player PlaylistInfo() failed: EOF
2024/07/22 01:41:59 01:41:59.530 WARNING player updateLibrary(): ListInfo() failed: write unix @->/home/steve/.config/mpd/socket: write: broken pipe
2024/07/22 01:41:59 01:41:59.530 WARNING player CurrentSong() failed: write unix @->/home/steve/.config/mpd/socket: write: broken pipe
2024/07/22 01:41:59 01:41:59.531 WARNING player PlaylistInfo() failed: write unix @->/home/steve/.config/mpd/socket: write: broken pipe
2024/07/22 01:41:59 01:41:59.531 WARNING player updateLibrary(): ListInfo() failed: write unix @->/home/steve/.config/mpd/socket: write: broken pipe
2024/07/22 01:41:59 01:41:59.531 WARNING player CurrentSong() failed: write unix @->/home/steve/.config/mpd/socket: write: broken pipe
2024/07/22 01:42:00 01:42:00.475 WARNING player Connection to MPD lost
2024/07/22 01:42:00 01:42:00.475 WARNING player Failed to connect to MPD: Status() failed: write unix @->/home/steve/.config/mpd/socket: write: broken pipe
```

**Comment by yktoo** [comment]
This surely looks like a networking issue. Can you try any other MPD client? `mpc` doesn't count as it only briefly establishes a connection, then drops it.

**Comment by StephenWetzel** [comment]
Thanks for your comment, I think I solved this.

GMPC was working, but both ymuse and quimup did not work, so I turned on verbose logging for mpd and looked at those logs while all three tried to connect.  The log while using GMPC looked normal, but when trying to use either ymuse or quimup I saw a lot of `client: error on client 110: Output buffer is full` messages.  Googling that a bit I found the option `max_output_buffer_size` in the `mpd.conf` file and set that to `32768`.  Restarting mpd and ymuse and it took a bit to load, but now seems to work (as well as quimup).
